README.md

```diff
@@ -12,29 +12,100 @@ Used to accept (or deny) the use of the private key(s) added to the SSH authenti
 ## Installation
 
 ### [Homebrew](https://brew.sh/)
-* Run:
 
-    ```
-    $ brew install theseal/ssh-askpass/ssh-askpass
-    ```
-* Follow caveats
+1. Run:
+
+   ```sh
+   brew install xquartz theseal/ssh-askpass/ssh-askpass
+   ```
+
+   See: [why install XQuartz?](#why-install-xquartz).
+
+1. Log out and log in again.
+
+1. Check that the `DISPLAY` environment variable is now set for `ssh-agent`
+   by XQuartz in "inherited environment":
+
+   ```sh
+   launchctl print gui/$UID/com.openssh.ssh-agent
+   ```
+
+1. On Apple Silicon Macs, run:
+
+   ```sh
+   sudo mkdir -p /private/var/select/X11/bin
+   sudo ln -s /opt/homebrew/bin/ssh-askpass /private/var/select/X11/bin/
+   ```
+
+   On Intel Macs, run:
+
+   ```sh
+   sudo mkdir -p /private/var/select/X11/bin
+   sudo ln -s /usr/local/bin/ssh-askpass /private/var/select/X11/bin/
+   ```
 
 ### [MacPorts](https://www.macports.org)
-* Install:
 
-    ```
-    $ sudo port install ssh-askpass
-    ```
+1. Install [XQuartz](https://www.xquartz.org/) from their packages.
+
+   MacPorts package this as well, but they've modified their install to
+   disable the user LaunchAgent by default. The upstream package just works.
+
+   See: [why install XQuartz?](#why-install-xquartz).
+
+1. Log out and log in again.
+
+1. Check that the `DISPLAY` environment variable is now set for `ssh-agent`
+   by XQuartz in "inherited environment":
+
+   ```sh
+   launchctl print gui/$UID/com.openssh.ssh-agent
+   ```
+
+1. Run:
+
+   ```sh
+   sudo port install ssh-askpass
+   sudo mkdir -p /private/var/select/X11/bin
+   sudo ln -s /usr/local/bin/ssh-askpass /private/var/select/X11/bin/
```

Comment:
MacPorts installs to `/opt/local`, as far as I'm aware.

Reply:
You're right, and it looks like `/opt/local` been that way for at least 10 years, so this is an easy fix.

I don't know why my install is in `/usr/local`... I probably changed it a long time ago.